Makefile

```diff
@@ -2,33 +2,62 @@
 
 .DEFAULT_GOAL = help
 
+GO-VERSION = 1.21.4
+GO-VER = go$(GO-VERSION)
+
+GO_OK := $(or $(USE_GO_CONTAINERS), $(shell which go 1>/dev/null 2>/dev/null; echo $$?))
+DOCKER_OK := $(shell which docker 1>/dev/null 2>/dev/null; echo $$?)
+
+ifeq ($(GO_OK), 0)  # use local go binary
+
+PAK_PATH=$(PWD)
+GO=go
+GOFMT=gofmt
+
+else ifeq ($(DOCKER_OK), 0)
+
+PAK_PATH=/brokerapi
```

Comment:
This looks like stuff copied from the makefile a CSB brokerpak. It could allow the tests to run in Docker. But do we want the complexity of that here? The argument with CSB brokerpaks is that someone who only knows Terraform HCL might want to make changes, and they might not want to install Go. I don't think that reasoning applies to `brokerapi`.

Reply:
The idea was to mount the folder into the container and use docker instead of the Go binary. I agree with you. I am complicating the solution. I just changed the code. Thank you very much